api/client/src/main/java/org/projectnessie/client/auth/oauth2/JacksonSerializers.java

```diff
@@ -54,8 +54,8 @@ public SecondsToInstantDeserializer() {
 
     @Override
     public Instant deserialize(JsonParser p, DeserializationContext ctxt) throws IOException {
-      int seconds = p.getValueAsInt(0);
-      if (seconds > 0) {
+      if (p.currentToken().isNumeric()) {
```

Comment:
Small improvement to parse any value, including zero and negative (zero happens sometimes).